demo/meeting/src/views/Meeting/index.tsx

```diff
@@ -26,7 +28,10 @@ const MeetingView = () => {
             className="videos"
             noRemoteVideoView={<MeetingDetails />}
           />
-          <MeetingControls />
+          {props.mode === MeetingMode.Spectator ?
+            <DynamicMeetingControls /> :
```

Comment:
Do we still need meeting control then? Cant we always use DynamicMeetingControls?

Reply:
We can't always use `DynamicMeetingControls`. The requirements for the correct use of this component are a bit strict - it works only if the device permission is unset after user've already joined the meeting. In any other cases, it can't transform. This is due to a limitation in our SDK, it would be great we could have a Provider to reveal the device permission status, but it's hard to do this now I think. Any suggestion?